src/App.js

```diff
@@ -1,26 +1,10 @@
-import React from 'react';
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
```

Comment:
## **WELL DONE 👌👍👏🙌🙂✅😉🙃🥳🎊🎉, it's is working as expected and even better I would say, good luck 🍀**